pkg/provisioner/helpers.go

```diff
@@ -151,10 +147,9 @@ func composeBucketName(obc *v1alpha1.ObjectBucketClaim) (string, error) {
 	}
 	bucketName := obc.Spec.BucketName
 	if bucketName == "" {
-		logD.Info("bucket name is empty, generating...")
 		bucketName = generateBucketName(obc.Spec.GenerateBucketName)
-		logD.Info("generated bucket name", "name", bucketName)
 	}
+	logD.Info("generated bucket name", "name", bucketName)
```

Comment:
The bucket name is not always generated. I think this change should be reverted...